tools/query_impc/impc_tool.xml

```diff
@@ -1,313 +1,311 @@
+        |Extract all phenotypes which are present in a particular gene set                                  |MP term ID, MP term name, genes                                                 |
+        +---------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+        |Extract images with a particular phenotype or a set of phenotypes                                  |External sample id, Gene symbol, Biological sample group, Sex, Colony id,       |
+        |                                                                                                   |Zygosity, Parameter name, Download url, Thumbnail url                           |
+        +---------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+        |Which IMPReSS parameters have been measured for a particular knockout                              |IMPReSS Parameter name                                                          |
+        +---------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+        |Which IMPRess parameters identified a significant finding for a particular knockout                |IMPReSS Parameter name, p-value                                                 |
+        +---------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+        |Full table of genes and all identified phenotypes                                                  |Gene accession id/Gene symbol, Identified phenotypes                            |
+        +---------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+        |Extract all genes names and ID measured in a specific IMPReSS pipeline                             |Gene accession id/Gene symbol, Gene name                                        |
+        +---------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+        |Extract all genes and corresponding phenotypes related to a particular top level phenotype category|Gene accession id/Gene symbol, Significant mp term id, Significant mp term name |
+        +---------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
 
-		|
-		|
+        |
+        |
 
-		**Credits**
+        **Credits**
```

Comment:
Please remove this from the help and use https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator-organization instead.